public/data/Pritam1812.json

```
@@ -0,0 +1,80 @@
+{
+    "name": "Pritam Hore",
+    "type": "personal",
+    "bio": "Aspiring Software Developer/Open Source Enthusiast!",
+    "avatar": "https://github.com/Pritam1812.png",
+    "links": [
+      {
+        "name": "Check me on GitHub!",
+        "url": "https://github.com/Pritam1812",
+        "icon": "github"
+      },
+      {
+        "name": "Connect with me on LinkedIn!",
+        "url": "https://www.linkedin.com/in/pritam1812",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Check out my Website!",
+        "url": "https://pritam1812.github.io/PortfolioWebsite",
+        "icon": "globe"
+      },
+      {
+        "name": "Add me on Facebook!",
+        "url": "https://www.facebook.com/Pritam1812",
+        "icon": "Facebook"
+      },
+      {
+        "name": "Ping me on Instagram!",
+        "url": "https://instagram.com/__pritam.___",
+        "icon": "Instagram"
+      },
+      {
+        "name": "Find me on Twitter!",
+        "url": "https://twitter.com/_Pritam_1812",
+        "icon": "twitter"
+      },
+      {
+        "name": "Join me on Discord!",
+        "url": "https://discord.com/users/496729273776865282",
+        "icon": "discord"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Learning C",
+        "date": "October 2020",
+        "icon": "Book",
+        "color": "blue",
+        "description": "My First Programming Langauge"
+      },
+      {
+        "title": "Started Algorithms and Data Structures",
+        "date": "December 2020",
+        "icon": "Youtube",
+        "color": "red",
+        "description": "Basics Of DSA in C"
+      },
+      {
+        "title": "Learning Java",
+        "date": "April 2021",
+        "icon": "LinkedIn",
+        "color": "purple",
+        "description": "Started Learning Java"
+      },
+      {
+        "title": "Learning HTML Essential Training",
+        "date": "November 2021",
+        "icon": "LinkedIn",
+        "color": "yellow",
+        "description": "Started Learning about Web Fondations"
+      },
+      {
+        "title": "Learning CSS Essential Training",
+        "date": "February 2022",
+        "icon": "LinkedIn",
+        "color": "green",
+        "description": "Started Learning about Web Designs"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```